packages/postgres-database/src/simcore_postgres_database/models/users.py

```diff
@@ -80,19 +73,29 @@ class UserStatus(Enum):
         "id",
         sa.BigInteger,
         nullable=False,
-        doc="Primary key for user identifier",
+        doc="Primary key index for user identifier",
     ),
     sa.Column(
         "name",
-        sa.String,
+        sa.String(),
         nullable=False,
-        doc="Display name. NOTE: this is NOT a user name since uniqueness is NOT guaranteed",
+        doc="username is a unique short user friendly identifier e.g. pcrespov, sanderegg, GitHK, ...",
+    ),
+    sa.Column(
+        "first_name",
+        sa.String(50),
+        doc="User's first name",
+    ),
+    sa.Column(
+        "last_name",
+        sa.String(50),
+        doc="User's last/family name",
     ),
     sa.Column(
         "email",
-        sa.String,
+        sa.String(50),
```

Comment:
stupid question: why do we limit at 50? will it not be cumbesome once you need to increase it? does it make such a huge difference in the postgres memory? aren't we getting the same issues like with docker network length of 64? well just my pepper here...

Reply:
Chatty told me to limit it to 50. Good API security practices also encourage to limit strings of identifiers. We do not need to write "el quijote" in an email, right :-) ? 

That gives for a good email `email.example.example@exampleexampleexample.com`

Reply:
This limit I took it from a suggestion of chatty. I decided to drop it in the database after reading more carefuly about character datatime with limits in postgres https://www.postgresql.org/docs/14/datatype-character.html and 
![Pasted image 20240113141711](https://github.com/ITISFoundation/osparc-simcore/assets/32402063/c5f26c51-61d7-4b1a-9e8e-2e745bed6fc2)


Instead, I am constraining these values when they are edited at the API to reasonable lengths. At the API level it is a security measure to limit strings. This way we have at least guarantees that nobody is storing "a full picture" in the first/last name columns. The suggested lenghts are set in the pydantic model that validates the input schemas for first/last names. See `FirstNameStr` and `LastNameStr`